app/server/models/user.py

```diff
@@ -149,6 +150,23 @@ class User(ManyOrgBase, ModelBase):
 
     exchanges = db.relationship("Exchange", backref="user")
 
+    @hybrid_property
+    def cashout_authorised(self):
+        # loop over all
+        # any_valid_token = [t.token for t in self.transfer_accounts]
+        ct = server.models.credit_transfer
+        example_transfer = ct.CreditTransfer(
+            transfer_type=ct.TransferTypeEnum.PAYMENT,
+            transfer_subtype=ct.TransferSubTypeEnum.AGENT_OUT,
+            sender_user=self,
+            recipient_user=self,
+            token=self.transfer_accounts[1].token,  # todo: should loop over all tokens
+            amount=0)
+
+        limits = example_transfer.get_transfer_limits()
```

Comment:
as per comment above, this is a nicer patern than doing anything in init.

Reply:
yes